app/controllers/admin/order_cycles_controller.rb

```diff
@@ -50,8 +50,11 @@ def update
       @order_cycle_form = OrderCycleForm.new(@order_cycle, params, spree_current_user)
 
       if @order_cycle_form.save
-        flash[:notice] = I18n.t(:order_cycles_update_notice) if params[:reloading] == '1'
-        render json: { :success => true }
+        respond_to do |format|
+          flash[:notice] = I18n.t(:order_cycles_update_notice) if params[:reloading] == '1'
+          format.html { redirect_to main_app.edit_admin_order_cycle_path(@order_cycle) }
```

Comment:
Aren't we still missing an HTML response in case it can't be saved?

Reply:
Probably, but our current `master` doesn't have one either, so this is not a regression.

The only time a html response is require is when the advanced settings form is used, and it currently only allows a single boolean attribute to be altered, so I think the prospect of an error message is pretty remote...